flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarListHandler.java

```diff
@@ -77,6 +82,11 @@ public boolean accept(File dir, String name) {
 						}
 					});
 
+					if (list == null) {
+						LOG.warn("Jar storage directory {} has been deleted.", jarDir.getAbsolutePath());
```

Comment:
Somehow this comment got lost before: Should we make the error message more explicit and say that it was `deleted externally` (e.g. not by Flink)?

Reply:
yes thats a good idea